### ISSUE TITLE
suggestion, create and poll on static file instead

### ISSUE THREAD

**Issue by andretsb** [jordanleven/force-refresh#716]
I think it might be better to have the plugin write out a static file that the clients poll, because right now having the polling request go through PHP and wordpress is a huge scalability problem, especially if your site is already a little slow. Static files however are much more light weight to serve.

Maybe easier to implement alternative to doing web sockets?

**Comment by jordanleven** [comment]
Yeah that's a good idea. WordPress does have some cacheing support on DB queries but writing to a static file would be quicker. For right now, WebSocket support is off the table since not all servers may be equipped to handle that.

If you're interested in adding support for storing versions as static versions, PRs are welcomed!
